R/bundlePackage.R

```diff
@@ -29,26 +29,41 @@ bundlePackages <- function(bundleDir,
   packages_list
 }
 
+usePackrat <- function() {
+  # Use RSCONNECT_PACKRAT when it has any value; fall-back to rsconnect.packrat when the environment
+  # variable is unset.
+  env_value <- Sys.getenv("RSCONNECT_PACKRAT", unset = NA)
+  if (is.na(env_value)) {
+    return(truthy(getOption("rsconnect.packrat", default = FALSE)))
+  }
+
+  return(truthy(env_value))
+}
```

Comment:
```suggestion
  value <- Sys.getenv("RSCONNECT_PACKRAT", unset = NA)
  if (is.na(value)) {
    value <- getOption("rsconnect.packrat", default = FALSE)
  }

  truthy(value)
}
```

That makes the logic a bit more clear to me.